client/api/README.md

```diff
@@ -178,6 +178,8 @@ From the CSI [proxy KEP](https://github.com/kubernetes/enhancements/blob/master/
 > 3. 0 releases if the API member is part of an Alpha/vNalpha1 version.
 > To continue running CSI Node Plugins that depend on an old version of csi-proxy.exe, vN, some of whose members have been removed, Kubernetes administrators will be required to run the latest version of the csi-proxy.exe (that will be used by CSI Node Plugins that use versions of CSIProxyService more recent than vN) along with an old version of csi-proxy.exe that does support vN.
 
+A given version of csi-proxy.exe will support multiple API versions, according to the deprecation schedule above. Running more than one version of csi-proxy.exe at one time is not supported yet.
```

Comment:
This conflicts with the sentence above, which talks about running multiple versions of the proxy

Reply:
Indeed... Removed!